contracts/JetStakingV2.sol

```diff
@@ -394,6 +401,7 @@ contract JetStakingV1 is AdminControlled {
     /// @param streamFundReceiver receives the rest of the reward tokens in the stream
     function removeStream(uint256 streamId, address streamFundReceiver)
         external
```

Comment:
while removing any stream, I see we check balance, but if balance is less than what should be transferred then shouldn't we store remaining amount? Or the `streamFundReceiver` cannot claim it afterwards as well?

Reply:
which balance you are referring to?

Reply:
I mean we check treasury balance and if its less than `releaseAuroraAmount` then we transfer what is there in treasury, but shouldn't we keep some kind of track for remaining amount? or that is lapsed if treasury does not have balance?

Reply:
As mentioned in the comment. This shouldn't happen, and if it is happen only due to round up of dust tokens (1^-18).